sieve_of_eratosthenes/sieveOfEratosthenes.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+#include <string.h>
+#include <math.h>     // for using square root 
+#include <stdbool.h>  // to include bool datatype 
+
+void sieveOfEratosthenes(int n) {
+	bool primes[n+1];
+	int i, sqrtOfn = sqrt(n)+1, j;
+	
+	memset(primes, true, sizeof(primes));
+	primes[0] = false;
+	primes[1] = false;
+
+	for (i = 2; i < sqrtOfn; i++) {
+		if (primes[i]) {
+			for (j = 2*i; j < n+1; j += i) {
+				primes[j] = false;
+			}
+		}
+	}
+
+	for (j = 2; j < n; j++) {
+		if (primes[j]) {
+			printf("%d ", j);
```

Comment:
@aashutoshrathi: I'm glad that you got it, and sorry that couldn't make you understand it in the first case. Need to work on those things :sweat_smile:.